### PR TITLE
Convert fast_ep to Python 3

### PR DESCRIPTION
**Pull request by rjgildea**

### CODE REVIEW

File: lib/generate_possible_spacegroups.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def test():
     assert(generate_chiral_spacegroups('P222') == \
            generate_chiral_spacegroups_unique('P222'))
 
-    print 'OK'
+    print('OK')
```
Comment (graeme-winter) [inline lib/generate_possible_spacegroups.py:85]: 🤦‍♂️

but yes

File: lib/generate_possible_spacegroups.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def test():
...
 
 if __name__ == '__main__':
```
Comment (graeme-winter) [inline lib/generate_possible_spacegroups.py:87]: Should ideally move these to actual tests?

File: lib/number_sites_estimate.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def test():
     number_sites = number_sites_estimate(cell, spacegroup)
     assert(number_sites < 1)
     assert(number_sites > 0)
-    print 'OK'
+    print('OK')
```
Comment (graeme-winter) [inline lib/number_sites_estimate.py:46]: Again, move to proper test? Or we do that another day.

File: src/fast_mr.py [hunk 4]
```diff
@@ -168,7 +168,7 @@ def do_mr(self):
         if cluster:
             pool.map(run_phaser_cluster, jobs)
         else:
-            print 1/0
+            print(1/0)
```
Comment (graeme-winter) [inline src/fast_mr.py:171]: Yeh that is really bad (I appreciate this was not added in this PR)

**Review by graeme-winter — APPROVED** [review]
I like the changes, thank you, but I really feel like we probably should go further with this and actually remove some of the old (GW implemented) cruft and clean up as we go. 

I don't mind doing this as part of this PR, or we could have a separate effort while this is fresh to dejunk / update the code base. I welcome your views.

**Comment by rjgildea** [comment]
> I like the changes, thank you, but I really feel like we probably should go further with this and actually remove some of the old (GW implemented) cruft and clean up as we go.
> 
> I don't mind doing this as part of this PR, or we could have a separate effort while this is fresh to dejunk / update the code base. I welcome your views.

Yes, I agree there is probably _a lot_ more changes/improvements we could make, but I think it probably better to make them as multiple smaller PRs, and just restrict this PR to the minimal necessary to get working with Python 3.

**Pull request merged**